package.json

```diff
@@ -32,8 +32,7 @@
     "eslint": "^2.7.0",
     "eslint-config-loopback": "^1.0.0",
     "express": "^4.12.0",
-    "loopback": "2.x.x",
-    "loopback-datasource-juggler": "2.x.x",
```

Comment:
>@loopback-team: *IMPORTANT: In LoopBack v3.0, loopback-datasource-juggler is no longer a peer dependency that has to be explicitly listed in app/project package.json. Keep this in mind while creating `loopback-2.x` branches and updating `master` to use `loopback@3`.*

Yea I did it based on @bajtos's comment.

Shouldn't I delete `loopback-datasource-juggler` here?